nncf/experimental/openvino_native/quantization/algorithms/fast_bias_correction/openvino_backend.py

```diff
@@ -78,13 +78,14 @@ def get_sub_input_output_names(subgraph: ov.Model) -> Tuple[str, str]:
         return subgraph.inputs[0].node.friendly_name, subgraph.outputs[0].node.friendly_name
 
     @staticmethod
-    def create_blob(shape: Tuple[int], data: List[np.ndarray], channel_axis: int) -> np.ndarray:
+    def create_input_data(shape: Tuple[int], data: List[np.ndarray], input_name: str, channel_axis: int) -> np.ndarray:
```

Comment:
Looks like return typehint should be updated

Reply:
updated